base/applications/regedit/main.c

```diff
@@ -176,25 +176,36 @@ void ExitInstance(HINSTANCE hInstance)
     UnloadAclUiDll();
 }
 
-BOOL TranslateChildTabMessage(PMSG msg)
+static BOOL InLabelEdit(HWND hWnd, UINT Msg)
 {
-    if (msg->message != WM_KEYDOWN) return FALSE;
-
-    /* Allow Ctrl+A on address bar */
-    if ((msg->hwnd == g_pChildWnd->hAddressBarWnd) &&
-        (msg->message == WM_KEYDOWN) &&
```

Comment:
Is the Ctrl-A now handled via the autocomplete thing?

Reply:
Ctrl+A is handled by ComCtl32 v6 Edit which I assume is supported everywhere?

Reply:
Our regedit has a correct manifest for comctl32 v6.